<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2740</width>
    <height>1684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2731</width>
      <height>1671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2740</width>
     <height>47</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_load_profile_file"/>
    <addaction name="action_manage_rules_repository"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_load_profile_file"/>
   <addaction name="action_manage_rules_repository"/>
   <addaction name="action_generate_rules"/>
  </widget>
  <action name="action_load_profile_file">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/coding.png</normaloff>../../Downloads/coding.png</iconset>
   </property>
   <property name="text">
    <string>Load profile file</string>
   </property>
   <property name="toolTip">
    <string>Open a .profile file from file system</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_manage_rules_repository">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/data-storage.png</normaloff>../../Downloads/data-storage.png</iconset>
   </property>
   <property name="text">
    <string>Manage Rules Repository</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_generate_rules">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/ui.png</normaloff>../../Downloads/ui.png</iconset>
   </property>
   <property name="text">
    <string>Generate Rules</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
